vimish-fold.el

```diff
@@ -131,6 +131,15 @@ overhead is undesirable."
   :type 'boolean
   :package-version '(vimish-fold . "0.2.3"))
 
+(defcustom vimish-fold-allow-nested nil
+  "Whether to allow nested folds
```

Comment:
The first line should end with a full stop. In `emacs-lisp-mode` Emacs should show warnings for these things I think...

Reply:
Oh I missed it. Done.

I don't have warning though, do you have something special for this in your configuration ?
(I don't have flycheck enabled right now though)

Reply:
Yeah, I think it's Flycheck.